src/lib/seam/components/DeviceTable/DeviceTable.tsx

```diff
@@ -109,9 +111,13 @@ export function DeviceTable({
     <div className={classNames('seam-device-table', className)}>
       <ContentHeader onBack={onBack} />
       <TableHeader>
-        <TableTitle>
-          {t.devices} <Caption>({filteredDevices.length})</Caption>
-        </TableTitle>
+        {title != null ? (
+          <TableTitle>
+            {title ?? t.devices} <Caption>({filteredDevices.length})</Caption>
+          </TableTitle>
+        ) : (
+          <div />
```

Comment:
Not sure if we want this. Preserves the location of the search field.

Reply:
Ah, yea good catch on the layout quirk. I don't love an empty div as a solution though, should be via class name if possible 🤔

Reply:
Using `<div className='seam-fragment' />` for now. If you want a different className and/or extracted to a atomic component, let me know!